md_api/src/main/java/com/wipro/ats/bdre/md/api/Lineage.java

```diff
@@ -45,6 +40,10 @@ public Lineage() {
     @Autowired
     LineageRelationDAO lineageRelationDAO;
 
+    public Lineage() {
+        AutowireCapableBeanFactory acbFactory = getAutowireCapableBeanFactory();
+        acbFactory.autowireBean(this);
+    }
     public Lineage execute(String[] params) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1161)